build/remote

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# This script should provision a remote linux server as an exit node
+# usage: PUBLIC_IP=165.227.241.194 ./build/remote root@165.227.241.194
```

Comment:
why are you using PUBLIC_IP if you are not using it?

Reply:
It's used below on line 27. e.g. you can provide it if the PUBLIC_IP is different from the ip/hostname you are SSHing to.

But it doesn't really need to be in this usage note. That's from before I made it use your SSH host as IP if no env var is provided.